src/main/java/org/broadinstitute/dsm/route/DownloadPDFRoute.java

```diff
@@ -44,100 +46,69 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DownloadPDFRoute.class);
 
-    private static final String SQL_SELECT_REALM_FOR_PARTICIPANT = "SELECT inst.instance_name, inst.base_url, inst.ddp_instance_id, inst.mr_attention_flag_d, " +
-            "inst.tissue_attention_flag_d, inst.es_participant_index, inst.es_activity_definition_index, inst.es_users_index, inst.auth0_token, " +
-            "inst.notification_recipients, inst.migrated_ddp, inst.billing_reference," +
-            " part.ddp_participant_id " +
-            "FROM ddp_participant part, ddp_instance inst WHERE inst.ddp_instance_id = part.ddp_instance_id " +
-            "AND part.ddp_participant_id = ?";
-
-    public static final String CONSENT_PDF = "/consentpdf";
-    public static final String RELEASE_PDF = "/releasepdf";
-    public static final String COVER_PDF = "/cover";
-    public static final String REQUEST_PDF = "/requestpdf";
     public static final String PDF = "/pdf";
+    public static final String BUNDLE = "/bundle";
 
+    private static final String COVER = "cover";
+    private static final String CONSENT = "consent";
+    private static final String RELEASE = "release";
+    private static final String TISSUE = "tissue";
+    private static final String IRB = "irb";
     private static final String JSON_START_DATE = "startDate";
     private static final String JSON_END_DATE = "endDate";
 
     @Override
     public Object processRequest(Request request, Response response, String userId) throws Exception {
         logger.info(request.url());
-        if (request.url().contains(RoutePath.DOWNLOAD_PDF)) {
-            String realm = null;
-            QueryParamsMap queryParams = request.queryMap();
-            if (queryParams.value(RoutePath.REALM) != null) {
-                realm = queryParams.get(RoutePath.REALM).value();
-            }
+        QueryParamsMap queryParams = request.queryMap();
```

Comment:
First of all I would create class something like `PDFPayload` which would handle payload from frontend since bunch of information comes from in request regarding pdf download.